Add details for support on this destination (#6033)
segmentio/segment-docs: src/connections/destinations/catalog/actions-pendo-web/index.md

[Pendo](http://www.pendo.io/){:target="_blank"} combines powerful software usage analytics with in-app guidance and user feedback capabilities, enabling even non-technical teams to deliver better product experiences to their customers or employees.

> This destination is maintained by Pendo. For issues with the Pendo Web (Actions) destination please reach out to [Pendo's support team](https://support.pendo.io/hc/en-us/articles/360034163971-Get-help-with-Pendo-from-Technical-Support)

Reviewer: ```suggestion
Pendo maintains this destination. For issues with the Pendo Web (Actions) destination, please reach out to [Pendo's support team](https://support.pendo.io/hc/en-us/articles/360034163971-Get-help-with-Pendo-from-Technical-Support){:target="_blank"}.
```
